utils/check_inits.py

```diff
@@ -83,9 +87,19 @@ def parse_init(init_file):
     while not lines[line_index].startswith("if TYPE_CHECKING"):
         # If the line is an if is_backend_available, we grab all objects associated.
         backend = find_backend(lines[line_index])
+        # Check if the backend declaration is inside a try block:
+        if _re_try.search(lines[line_index - 1]) is None:
+            backend = None
+
```

Comment:
This actually makes the check not check anything anymore since all the lines with an `if is_xxx_available` are now in a `Try` block.